oracle/learning-library
# Kay's fixes

QA Verification:
 
Switch Introduction and Getting Started Labs    = DONE
Introduction
•	This concludes introduction. → This concludes the introduction    	= DONE
•	Take out the quotes around title and use commas or parens    		= DONE
Prerequisites  - done changed to LAB 0
•	The Prerequisites doesn't make sense here, it should be  Lab.  Make it Lab 0 if you need it    = DONE
Lab 1 - Environment Setup
•	Resource manager would do better here instead of terraform.  It's a much better experience for the user.  See one of the Enterprise Manager workshops or talk to Rene Fontcha about better options for implementing        = Agreed with Kay, this is optional, not necessarily.
Lab 2 - Prepare Target ATP
•	Step 1, menu for Oracle Cloud has changed, change your instructions  = OCI theme has reverted back to original.
Everything else looks great.  Make these changes and contact Anoosha Pilli on my team, she will turn this on in production for you.  Lets get this into production and on to the main page as a featured workshop!
